app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -659,6 +659,10 @@ class TabTrayView(
                 1
 
             layoutManager?.scrollToPosition(recyclerViewIndex)
+            smoothScrollBy(
```

Comment:
We should avoid scrolling if the first item is selected.

Reply:
I have reversed the scrolling and removed the `+1`.
So, we scroll to the selected item, then scroll "down" so half of the item above will be visible. If there is no item above, the scroll would not happen (there is not scroll space left).